<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightSearchWindow</class>
 <widget class="QWidget" name="FlightSearchWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Available Flights</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>701</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>801</width>
      <height>561</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font-size: 50px;
color:black;
font-weight:bold;</string>
       </property>
       <property name="text">
        <string> Search Your Desired Flight Here!</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Origin:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="originLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Destination:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="destinationLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Departure Date:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>Search Flights</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="flightsTableWidget"/>
     </item>
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>20</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
